crypto/threads_win.c

```diff
@@ -103,6 +103,7 @@ struct rcu_lock_st {
     CRYPTO_CONDVAR *alloc_signal;
     CRYPTO_MUTEX *prior_lock;
     CRYPTO_CONDVAR *prior_signal;
+    CRYPTO_RWLOCK * rw_lock;
```

Comment:
There should really not be a space after the `*` here

Reply:
Fixed